### ISSUE TITLE
Autostart

### ISSUE THREAD

**Issue by cenobite1** [mhogomchungu/sirikali#15]
How do i remove sirikali from autostart?
Thanks

**Comment by mhogomchungu** [comment]
SiriKali does not have any functionality that causes it to auto start.

I can think of two causes of it doing that:
1. It is packaged by somebody else(your distro??) and they gave it that ability.
2. Your have session restore feature enabled and you desktop auto starts it as part of session restoration process when you log in.

What distribution are you using? 
What desktop environment are you using?
How did you install SiriKali?

**Comment by cenobite1** [comment]
Hello,
Thanks for the prompt response.
I use KDE (Kubuntu) and i use the session restore functionality, but i made sure i exited sirikali before i did it, in order to make sure that sirikali wouldn't start.
I compiled sirikali from sources.

**Comment by mhogomchungu** [comment]
How did you exit it? Clicking the close button on main window does not close the application,it just minimizes it. To close it,you have to go through menu->quit or tray icon-quit.

**Comment by cenobite1** [comment]
I did it via menu->quit.

**Comment by mhogomchungu** [comment]
If you closed it before you log out,then i do not see how it will get auto started on next logon, will test later on today in my KDE installation to see if i can reproduce the problem but the problem here is most likely not in SiriKali.

**Comment by mhogomchungu** [comment]
Just curious,you use SiriKali as a front end to what backend? encfs,cryptfs,gocryptfs,securefs or ecryptfs?

**Comment by cenobite1** [comment]
I use it for all the above backends ( i didnt know ecryptfs was already supported, i will check that out).
I encrypt mostly using gocryptfs (the author has a nice comparative review of all crypt fs solutions on his webpage).
Regarding the application behaviour, in my experience other apps behave correct eg Krusader, dolphin, Kate, etc...they all stay removed at next startup when i save session (after having closed them), i just verified it as we speak.

**Comment by mhogomchungu** [comment]
I tested here and it worked fine.

I configured my desktop to restore last session,logged out while sirikali was running and it was auto started at log in.

I configured my desktop to start with an empty session,logged out and logged back in and sirikali did not start.

I tried to look around where applications that are to be auto started at session restoration time are stored but didnt find anything.

When did you install sirikali and when did you start noticing the problem? are you using activities? maybe they are involved somehow..
